src/components/gitOps/GitOpsConfiguration.tsx

```diff
@@ -451,8 +451,8 @@ class GitOpsConfiguration extends Component<GitOpsProps, GitOpsState> {
                 </div>
 
                 <div className="form__buttons">
-                    <button type="submit" disabled={this.state.saveLoading} onClick={(e) => { e.preventDefault(); this.saveGitOps() }} tabIndex={5} className="cta">
-                        {this.state.saveLoading ? <Progressing /> : "Save"}
+                    <button type="submit" disabled={this.state.saveLoading} onClick={(e) => { e.preventDefault(); this.saveGitOps() }} tabIndex={5} className={`cta ${this.state.saveLoading ? 'cursor-not-allowed': '' }`}>
```

Comment:
are we removing the loader from save? In case of user clicking save it should be visible

Reply:
As per discussion, changes have been made